lib/config/presets/internal/regex-managers.ts

```diff
@@ -41,4 +41,16 @@ export const presets: Record<string, Preset> = {
       },
     ],
   },
+  tfvarsVersions: {
+    description: 'Update `*_version` variables in `.tfvars` files',
```

Comment:
```suggestion
    description: 'Update `*_version` variables in `.tfvars` files.',
```

Let's end the sentence with a full stop.

I'll let the maintainers review the code. 🙂